features/data_converter/binary/feature.py

```diff
@@ -0,0 +1,40 @@
+import os
+import json
+
+from temporalio import workflow
+from temporalio.api.enums.v1 import EventType
+from temporalio.client import WorkflowHandle
+
+from harness.python.feature import load_json_payload, to_json_payload, register_feature, Runner
+
+
+@workflow.defn
+class Workflow:
+    """
+    run a workflow that returns binary value `0xdeadbeef`
+    """
+    @workflow.run
+    async def run(self) -> bytes:
+        return bytes.fromhex("deadbeef")
+
+
+async def check_result(runner: Runner, handle: WorkflowHandle) -> None:
+    # verify client result is binary `0xdeadbeef`
+    result = await handle.result()
+    assert result == bytes.fromhex("deadbeef")
+
+    # get result payload of WorkflowExecutionCompleted event from workflow history
+    history = await runner.get_history_events(handle)
+    e = next(e for e in history if e.event_type == EventType.EVENT_TYPE_WORKFLOW_EXECUTION_COMPLETED)
+    payload = e.workflow_execution_completed_event_attributes.result.payloads[0]
+
+    # load JSON payload from `./payload.json` and compare it to JSON representation of result payload
+    with open(os.path.join(os.path.dirname(__file__), 'payload.json'), encoding='ascii') as f:
```

Comment:
`runner.feature.file` is available

Reply:
sgtm

Reply:
Not sure what this gives me, `__file__` is shorter.

Reply:
It is consistent regardless of what helper file you may be in. Yes technically here you can use `__file__` and other features you can use the one we provide on runner if it's important to save some characters.

Reply:
I'd consider this a nit but sure